user_profile/managers.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import Manager
+from django.contrib.sites.models import Site
+
+
+class UserProfileManager(Manager):
+
+    def total_profiles(self):
+        return self.count()
+
+
+class SettingsManager(Manager):
+
+    def  get_site_id(self, site_domain):
+        current_domain = Site.objects.get_by_natural_key(site_domain)
+        return current_domain.id
+
+    def get_current_settings(self, current_site_id):
```

Comment:
both `get_site_id ` and  `get_current_settings` can be merged into a single method  something like 
https://docs.djangoproject.com/en/2.2/ref/contrib/sites/#django.contrib.sites.shortcuts.get_current_site
```
def get_settings_by_site(request)
current_site = get_current_site(request)
return self.get(site=current_site)
```

Reply:
done